database.yml

```diff
@@ -50,6 +53,20 @@
       vars:
         db_owner: "{{ authoring_db_user }}"
         db_name: "{{ authoring_db_name }}"
+    - include: tasks/create_db_user.yml
+      when: XXX
+      vars:
+        db_user: "replicator"
   tags:
     - database
     - be
+
+- name: set up replication on replicant
+  hosts:
+    - replicant
+  roles:
+    - pg_replicant
```

Comment:
Any chance you can change the role name to `postgres_replicant`? This would then share the name with existing postgres role. It may help to keep things organized. Just a thought...